packages/penrose-core/src/utils/CollectLabels.ts

```diff
@@ -1,4 +1,5 @@
 import memoize from "fast-memoize";
+import { Shape } from "types/shapeTypes";
 const mathjax = require("mathjax-full/js/mathjax.js").mathjax;
```

Comment:
Can we fix these imports? I tried to create an UMD target, but rollup is not happy with these `require`s mixed in